components/elements/inner/ContentManagerSubMenu.js

```diff
@@ -59,11 +59,10 @@ const ContentManagerSubMenu = ({title, currentTypeSlug}) => {
     useEffect(() => { 
       (async () => {
 
-        await loadEntityTypes({rootState, rootDispatch, onStartLoad, onEndLoad});
+        await loadEntityTypes({rootState, rootDispatch, onStartLoad, onEndLoad, currentTypeSlug});
```

Comment:
`loadEntityTypes`, as the name suggests, is intended to load all of the entity types from the database. Adding an extra parameter makes this function deviate from its original intention. I think the handling of the current slug can be handled differently (see other code comments).